addons/event/i18n/event.pot

```diff
@@ -60,7 +60,7 @@ msgstr ""
 #: model:ir.actions.report,print_report_name:event.action_report_event_registration_foldable_badge
 msgid ""
 "'Foldable Badge - %s - %s' % ((object.event_id.name or "
-"'Event').replace('/',''), object.name.replace('/',''))"
+"'Event').replace('/',''), (object.name or '').replace('/',''))"
```

Comment:
I know the one who did that change :disguised_face: forgot to do it when fixing the report name :D

Reply:
should I still make this change or move it back to the way it was before?